src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -82,61 +68,56 @@ private async void Page_Loaded(object sender, RoutedEventArgs e)
 		{
 			AppSettings.ThemeModeChanged += AppSettings_ThemeModeChanged;
 
-			if (_usingWinUI)
+			if (FilePropertiesHelpers.IsWinUI3)
```

Comment:
Can this ever be false?

Reply:
Removed WinUI3 checking logics in this page